projects/frontend/docker-compose.yml

```diff
@@ -47,6 +48,7 @@ services:
       GOVUK_PROXY_STATIC_ENABLED: "true"
       PLEK_SERVICE_CONTENT_STORE_URI: https://www.gov.uk/api
       PLEK_SERVICE_STATIC_URI: https://assets.publishing.service.gov.uk
+      PLEK_SERVICE_SEARCH_API_URI: https://www.gov.uk/api
```

Comment:
If you're setting this you probably don't need to add the `Makefile` or app dependency bits?

Reply:
I thought I'd add it for consistency, in case someone does try to run all the things locally. 😬

Reply:
I think the dependency will be required, if you run the app without specifying live or integration (i.e. to use the local apps only).